builtin/settingtypes.txt

```diff
@@ -2054,16 +2054,16 @@ liquid_update (Liquid update tick) float 1.0 0.001
 #    Stated in mapblocks (16 nodes).
 block_send_optimize_distance (Block send optimize distance) int 4 2 32767
 
-#    If enabled the server will perform map block occlusion culling based on
+#    If enabled, the server will perform map block occlusion culling based on
 #    on the eye position of the player. This can reduce the number of blocks
-#    sent to the client 50-80%. The client will not longer receive most invisible
-#    so that the utility of noclip mode is reduced.
-server_side_occlusion_culling (Server side occlusion culling) bool true
+#    sent to the client 50-80%. The client will no longer receive most invisible
+#    blocks, so that the utility of noclip mode is reduced.
```

Comment:
```suggestion
#    sent to the client by 50-80%. Clients will no longer receive most
#    invisible blocks, so that the utility of noclip mode is reduced.
```